### PR TITLE
Syntax Highlighting

### LINKED ISSUE #26: Code blocks do not work for languages besides scratch
Normally in markdown, in places like github / discord, code blocks will automatically highlight syntax,
```js
like(this);
```
However, on flagclicked this does not work.

### PR DESCRIPTION
**Pull request by CubeyTheCube**
Resolves #26 

I added a [syntax highlighting library](http://rainbowco.de/) to highlight non-scratch code. Just in case we were going to add this functionality to Flag Clicked. It worked when I tested it, but alert me about any issues.

### CODE REVIEW

**Comment by CubeyTheCube** [comment]
ok for the record, i have no idea why that check failed. it works in repl.it

File: pages/new/index.html [hunk 1]
```diff
@@ -29,13 +29,28 @@
         },
         computed: {
           compiledMarkdown() {
-            setTimeout(() => {
```
Comment (TheColaber) [inline pages/new/index.html:32]: Why remove the setTimout... iirc after Vue rerenders it should be after 0 milliseconds.

Comment (CubeyTheCube) [reply pages/new/index.html:32]: @TheColaber Using requestAnimationFrame is better practice and more optimized; before the scratchblocks used to flash but now they don't.

Comment (TheColaber) [reply pages/new/index.html:32]: Seems to be blinking more for me...

https://user-images.githubusercontent.com/72760579/112247374-8f8f9980-8c2a-11eb-8285-33ea54c72c83.mp4

Comment (TheColaber) [reply pages/new/index.html:32]: huh, but the scratchblocks don't blink at all... interesting...

Comment (markverb1) [reply pages/new/index.html:32]: > huh, but the scratchblocks don't blink at all... interesting...

because scratchblocks aren't really code blocks

Comment (TheColaber) [reply pages/new/index.html:32]: @CubeyTheCube Seems like the function itself... `Rainbow.color();` is causing the blinking... do you know how to disable it and/or pick a different one?

**Comment by TheColaber** [comment]
Merging for now... any PRs for a nonblinking language would be great.

**Pull request merged**